src/test/java/com/puppycrawl/tools/checkstyle/utils/TokenUtilTest.java

```diff
@@ -277,4 +278,20 @@ public void testForEachChild() {
         assertEquals(secondSibling, firstChild, "Mismatched child node");
     }
 
+    @Test
+    public void testIsTypeDeclaration() {
```

Comment:
what force you do this test ?
We need to test all real inputs. If something is missed it is better to extend Inputs.

Reply:
Adding `testIsTypeDeclaration()` to TokenUtil reduces coverage by 0.03, so I added this test.  What inputs should I extend in this case to restore coverage?

Reply:
Maybe a case where it makes sense skipping the test? Either way seems ok to me

Reply:
@esilkensen I'm just not sure how else to test this with actual code, since none of the other methods in TokenUtilTest.java use real input.  Until I use this method more (likely in the records check updates), I don't think it is getting exercised enough with real input to get enough coverage.

Reply:
I call back my point .... I do not know how resolve it.
It is good to use Util methods, but in the same time we use pitest split to speedup pitest execution and this force us to write non real test cases. So if new Check start to use this method we will miss a big chunk of test that Check should have to cover cases of different type.

Probably I should demand to copy all Util methods in code of Check on first implmentation, make sure that pitest is happy and only then use Util methods.
Or we should think on to move Utils pitest in same pitest execution as Checks.